src/lt/objs/deploy.cljs

```diff
@@ -85,38 +118,145 @@
                             (throw (js/Error. (str "Error downloading: " from " status code: " (.-statusCode resp)))))))
         (.pipe out))))
 
+(def temp-download-tar-path (str home-path "/tmp" tar-extension))
+
+; TODO: Rename to `download-release-file`?
 (defn download-zip [ver cb]
   (let [n (notifos/working (str "Downloading version " ver " .."))]
-    (download-file (tar-path ver) (str home-path "/tmp.tar.gz") (fn [e r body]
-                                                                  (notifos/done-working)
-                                                                  (cb e r body)))))
+    (tar-path ver
+              (fn [url]
+                (console/log url)
```

Comment:
Well, yes *and* no :smile: 

This is definitely debris leftover from my hacking this together.

I imagine no one will really find the URL being logged to their console to be of much utility so I'll take it out when I get around to cleaning up this commit.